shared/src/main/scala/com/redi/scalademo/business/NumericStringValidator.scala

```diff
@@ -5,6 +5,7 @@ import scala.util.Try
 class NumericStringValidator {
 
   def isValidNumber(string: String): Boolean = {
+    println(s"Validating if '$string' is a valid number")
```

Comment:
I thought, in addition to network log, printing when the validation is happening is a good way to demo where the validation is actually happening.